packages/svelte/src/lib/component/UploadDropzone.svelte

```diff
@@ -0,0 +1,128 @@
+<script lang="ts" generics="TRouter extends FileRouter">
+  import type { FileWithPath } from "file-selector";
+
+  import { classNames, generateClientDropzoneAccept } from "uploadthing/client";
+  import type { FileRouter } from "uploadthing/server";
+
+  import { INTERNAL_uploadthingHookGen } from "../useUploadThing";
+  import Dropzone from "./Dropzone.svelte";
+  import {
+    allowedContentTextLabelGenerator,
+    generatePermittedFileTypes,
+    progressHeights,
+    type UploadthingComponentProps,
```

Comment:
separate type imports preferred. I thought I had a lint rule for this 🤔

Reply:
Let me check if there's an issue with linting in svelte files

Reply:
Ok so I had to change `eslint-plugin-svelte3` cause it isn't compatible with svelte 4, the new one is `eslint-plugin-svelte`.

But, I had to put the svelte eslint overrides in the `.eslintrc.cjs` file, cause it does not work if I put it in the eslint config package.

Also, the new plugin does not detect generics in svelte components, so I had to put a bunch of eslint-disable comments, which is... unfortunate. There's already an [issue](https://github.com/sveltejs/svelte-eslint-parser/issues/306) for it